framework/src/modules/chain/blocks/transactions/votes_weight.js

```diff
@@ -93,7 +93,7 @@ const revertVotedDelegatePublicKeys = (
 const updateRecipientDelegateVotes = (
 	stateStore,
 	transaction,
-	undo = false,
+	isUndo = false,
```

Comment:
I think `undo` is already action, so it's fine as it was

Reply:
we have another undo variable usage in the same function for a different purpose, so it was shadowing it since it was used as a boolean field, I just named it isUndo. Let's discuss in person for more clarity and same for the above.